Samples.xcodeproj/project.pbxproj

```diff
@@ -328,6 +328,8 @@
 		955AFAC62C110B8A009C8FE5 /* ApplyMosaicRuleToRastersView.swift in Copy Source Code Files */ = {isa = PBXBuildFile; fileRef = 955AFAC32C10FD6F009C8FE5 /* ApplyMosaicRuleToRastersView.swift */; };
 		9579FCEA2C3360BB00FC8A1D /* EditFeatureAttachmentsView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9579FCE92C3360BB00FC8A1D /* EditFeatureAttachmentsView.swift */; };
 		9579FCEC2C33616B00FC8A1D /* EditFeatureAttachmentsView.swift in Copy Source Code Files */ = {isa = PBXBuildFile; fileRef = 9579FCE92C3360BB00FC8A1D /* EditFeatureAttachmentsView.swift */; };
+		957B25D42E31BE27007E88F9 /* ShowWfsLayerWithXmlQueryView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 957B25D32E31BE23007E88F9 /* ShowWfsLayerWithXmlQueryView.swift */; };
```

Comment:
Can you clean up the Xcode project file? The filename mismatches the entries in project file, i.e., `ShowWfsLayerWithXmlQueryView` vs `ShowWFSLayerWithXMLQueryView`. The latter is correct.